example/particles/main.cpp

```diff
@@ -34,8 +30,23 @@ int main(int argc, char *argv[]) {
     pman.ParthenonFinalize();
     return 1;
   }
-  // Now that ParthenonInit has been called and setup succeeded, the code can now
-  // make use of MPI and Kokkos
+
+  // Redefine parthenon defaults
+  pman.app_input->ProcessPackages = particles_example::ProcessPackages;
+  pman.app_input->ProblemGenerator = particles_example::ProblemGenerator;
+  if (pman.pinput->GetString("parthenon/mesh", "ix1_bc") == "user") {
+    pman.app_input->boundary_conditions[parthenon::BoundaryFace::inner_x1] =
+        parthenon::BoundaryFunction::OutflowInnerX1;
```

Comment:
How about adding a comment here that setting the (normal)  boundary conditions like this is just done here to make the hardcoded example work and is not considered best practice (because this way the info in the parameter file and outputs does not reflect what's actually happening in the code and boundary conditions are not aligned, i.e., mixing standard and user).

Reply:
I actually think this is a reasonable use case -- one could imagine wanting outflow boundary conditions for a simulation, but the default parthenon swarm boundary outflow conditions of just deleting the particles as they leave not being suitable. A better particle boundary condition is often having a particle enter the simulation to match each particle leaving the simulation, assuming some equilibrium distribution between particles and cell variables at the boundary. I did add a comment here though going into this in some more detail -- let me know if you don't think that's sufficient.

Reply:
I see. I looked at it from a point of view where normal variables and swarms were treated separately (and therefore overwriting the normal variables boundary conditions felt odd).
Now, I understand that the "`user`" part actually applies to variables and swarms and in the example the "`user`" condition just reused one of the default implementations by chance -- which makes sense.
Thanks for adding the comment.